CMakeLists.txt

```diff
@@ -32,12 +32,12 @@ SET(PROJECT_DESCRIPTION "${PROJECT_NAME_UCASE} - Free music production software"
 SET(PROJECT_COPYRIGHT   "2008-${PROJECT_YEAR} ${PROJECT_AUTHOR}")
 SET(VERSION_MAJOR       "1")
 SET(VERSION_MINOR       "1")
-SET(VERSION_PATCH       "91")
-#SET(VERSION_SUFFIX     "")
-SET(VERSION             "${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_PATCH}")
-IF(VERSION_SUFFIX)
-	SET(VERSION     "${VERSION}-${VERSION_SUFFIX}")
-ENDIF(VERSION_SUFFIX)
+SET(VERSION_BUILD       "91")
```

Comment:
It is `1.2.0-rc2` now.

The commit following `1.2.0` should be tagged `1.3.0-alpha` to not break `git describe` and development versions, since `1.3.0-alpha.x` is greater than the stable update `1.2.1`. `1.2.0.x` versions should be reserved for the preparation of `1.2.1` in the `stable-1.2` branch.